src/main/java/com/amazon/ion/impl/bin/IonEncoder_1_1.java

```diff
@@ -129,7 +128,7 @@ public static int writeIntValue(WriteBuffer buffer, final BigInteger value) {
      * Writes a float to the given WriteBuffer using the Ion 1.1 encoding for Ion Floats.
      * @return the number of bytes written
      */
-    public static int writeFloat(WriteBuffer buffer, final float value) {
+    public static int writeFloatValue(WriteBuffer buffer, final float value) {
```

Comment:
🗺️ This was renamed to `write___Value` to be consistent with the other methods in this class.